src/create-reminder-signup/lib/identity.ts

```diff
@@ -7,6 +7,9 @@ import {
 	success,
 } from '../../lib/identity';
 import { ReminderStage } from '../lambda/models';
+import { fetchWithRetry } from './fetch-with-retry';
+
+const IDENTITY_ACCOUNT_CREATION_MAX_RETRIES = 1;
```

Comment:
We can obviously up this, but I was just curious how much a single retry would help